lib/explorer/polars_backend/shared.ex

```diff
@@ -149,7 +149,7 @@ defmodule Explorer.PolarsBackend.Shared do
   def internal_from_dtype(:integer), do: "i64"
   def internal_from_dtype(:string), do: "str"
 
-  defp error_message({_err_type, error}) when is_binary(error), do: error
+  defp error_message(error) when is_binary(error), do: error
   defp error_message(error), do: inspect(error)
```

Comment:
Will this branch ever be called?

Reply:
I don't think so, but if I remove this branch the function will be:

```elixir
defp error_message(error), do: error
```

So is it better to remove it?

Reply:
Remove the branch but keep the guard in the previous one?

Reply:
https://github.com/elixir-nx/explorer/pull/623/commits/cafb699044a648a8a3d07c650cecaafc309d6514